locale/ForemanDocker.pot

```diff
@@ -0,0 +1,633 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: version 0.0.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-06-01 12:43-0400\n"
+"PO-Revision-Date: 2015-06-01 12:43-0400\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: ../foreman-docker/app/models/docker_registry.rb:34
+msgid "Docker/Registry"
+msgstr ""
+
+#: ../foreman-docker/app/models/service/containers.rb:99
+msgid "Could not start container"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/models/concerns/fog_extensions/fogdocker/server.rb:39
+msgid "%{cores} Cores and %{memory} memory"
+msgstr ""
+
+#: ../foreman-docker/app/models/container.rb:60
+msgid "Docker/Container"
+msgstr ""
+
+#: ../foreman-docker/app/models/foreman_docker/docker.rb:138
+msgid "Error creating communicating with Docker. Check the Foreman logs: %s"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:22
+msgid "List all docker registries"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:29
+msgid "Show a docker registry"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:34
+msgid "Create a docker registry"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:41
+msgid "Update a docker registry"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:48
+msgid "Delete a docker registry"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:13
+msgid "List all containers"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:15
+msgid "List all containers in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:29
+msgid "Show a container"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:31
+msgid "Show container in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:43
+msgid "Registry this container will have to\\n                                                        use to get the image"
```

Comment:
whitespace is very broken here, perhaps needs to use heredoc style in the source?  (Sorry, haven't checked)